doc/man/SConscript

```diff
@@ -0,0 +1,11 @@
+"""Build DAOS man pages"""
+import os
+
+def scons():
+    """Execute build"""
+    Import('env')
+
+    env.Install("$PREFIX/usr/share/man/", ['man8'])
```

Comment:
I think this only installs the directory, and you need to specify the actual files to be installed. Looks like the RPM build failed because the manpages weren't copied to the right place.

Reply:
It will install everything under the directory. At least that is what I did when I typed scons install. Now the question is whether that path I have there is what the rpm spec has in %_mandir. I'll have to check that out.